src/Symfony/Component/VarExporter/LazyGhostTrait.php

```diff
@@ -223,6 +223,10 @@ public function &__get($name): mixed
 
                 return $accessor['get']($this, $name, null !== $readonlyScope);
             } catch (\Error) {
+                if (preg_match('/^Cannot access uninitialized non-nullable property ([\w\\\\@]+::\$\w+) by reference$/', $e->getMessage(), $matches)) {
+                    throw new \Error(sprintf('Typed property %s must not be accessed before initialization', $matches[1]), $e->getCode(), $e->getPrevious());
```

Comment:
we should use concatenation to please fabbot:
```suggestion
                    throw new \Error('Typed property '.$matches[1].' must not be accessed before initialization', $e->getCode(), $e->getPrevious());
```